### DISCUSSION TITLE
Welcome to DHNx Discussions!

### DISCUSSION THREAD

**Discussion by joroeder** [oemof/DHNx#98] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gitfeldhase** [comment]
Hi everyone,

I am a masterstudent writing my thesis on DHNs and will use the simulation toolbox of dhnx.
Since I am a very beginner with oemof and dhnx, I am looking forward for any kind of support and willing to contribute if possible.

Best regards

Lars
